Sources/Plasma/FeatureLib/pfMetalPipeline/plMetalDevice.h

```diff
@@ -259,20 +259,30 @@ class plMetalDevice
     
     void LoadLibrary();
 
+    void BeginNewRenderPass();
+    void ReleaseSamplerStates();
+    void ReleaseFramebufferObjects();
+
+    // Blur states
+    std::unordered_map<float, NS::Object*> fBlurShaders;
+    
+    // MARK: - Post processing
+private:
     bool NeedsPostprocessing() const
     {
         return fGammaLUTTexture != nullptr;
     }
+    void                      PreparePostProcessing();
+    void                      FinalizePostProcessing();
     void                      PostprocessIntoDrawable();
     void                      CreateGammaAdjustState();
     MTL::RenderPipelineState* fGammaAdjustState;
 
-    void BeginNewRenderPass();
-    void ReleaseSamplerStates();
-    void ReleaseFramebufferObjects();
-
-    // Blur states
-    std::unordered_map<float, NS::Object*> fBlurShaders;
+    // MARK:  - Device capabilities
+private:
+    /// Returns true if the device supports tile memory features such as directly writable render buffers.
+    inline BOOL SupportsTileMemory() const { return fSupportsTileMemory; }
+    BOOL fSupportsTileMemory;
```

Comment:
This should be a `bool` rather than a `BOOL`

MTL::Device `supportsFamily` returns a `bool` in C++: https://github.com/bkaradzic/metal-cpp/blob/a63bd172ddcba73a3d87ca32032b66ad41ddb9a6/Metal/MTLDevice.hpp#L388